tests/shardddl3/run.sh

```diff
@@ -1020,20 +1020,24 @@ function DM_DropAddColumn_CASE() {
 
     restart_master_on_pos $reset "1"
 
-    run_sql_source1 "alter table ${shardddl1}.${tb1} drop column b;"
-    run_sql_source1 "insert into ${shardddl1}.${tb1} values(4);"
-
-    restart_master_on_pos $reset "2"
-
     run_sql_source2 "alter table ${shardddl1}.${tb1} drop column c;"
-    run_sql_source2 "insert into ${shardddl1}.${tb1} values(5,5);"
+    run_sql_source2 "insert into ${shardddl1}.${tb1} values(4,4);"
 
-    restart_master_on_pos $reset "3"
+    restart_master_on_pos $reset "2"
 
     # make sure column c is fully dropped in the downstream
     check_log_contain_with_retry 'finish to handle ddls in optimistic shard mode' $WORK_DIR/worker1/log/dm-worker.log
     check_log_contain_with_retry 'finish to handle ddls in optimistic shard mode' $WORK_DIR/worker2/log/dm-worker.log
 
+    run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
```

Comment:
could we remove above `check_log_contain_with_retry`?

Reply:
seems no, `no DDL lock exists` doesn't guarantee that master received info sync `drop column c` to the downstream, probably master didn't received  any ddl infos.